src/lib/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ target_include_directories(
 
 install(TARGETS openstudiolib
         EXPORT openstudio
-        DESTINATION lib
+        DESTINATION ${LIB_DESTINATION_DIR}
```

Comment:
libopenstudiolib destination: on windows it must go to the bin/ folder now that openstudio.exe isn't statically linked and requires it.